src/pages/Dashboard.vue

```diff
@@ -247,8 +217,58 @@
                 </div>
               </div>
             </div>
+            <div class="mt-4">
+              <div class="bg-white py-4 border-gray-300 border-2 rounded-lg">
+                <!-- number of questions answered -->
+                <div
+                  v-if="pending"
+                  class="w-full flex flex-col justify-center animate-pulse py-4"
+                >
+                  <p class="h-10 w-1/2 place-self-center bg-yellow-900 rounded-md"></p>
+                  <p
+                    class="h-4 w-1/2 place-self-center bg-yellow-900 rounded-md mt-1"
+                  ></p>
+                </div>
+                <div v-else>
+                  <!-- value -->
+                  <p
+                    :class="getCardMetricValueClass(numQuestionsAnswered)"
+                    data-test="questionAnswered"
+                  >
+                    {{ numQuestionsAnswered }}
+                  </p>
+                  <div :class="cardMetricTitleClass">
+                    <!-- title -->
+                    <p>
+                      {{ $t("dashboard.summary.num_questions_answered.title") }}
+                    </p>
+                    <div
+                      class="flex relative"
+                      v-tooltip="$t('tooltip.dashboard.summary.num_questions_answered')"
+                    >
+                      <!-- info icon -->
+                      <inline-svg
+                        :src="questionIcon"
+                        class="h-4 w-4 text-yellow-900 fill-current hover:cursor-pointer"
+                      ></inline-svg>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
           </div>
         </div>
+        <div
+          v-if="hasSurveyQuestion"
+          class="pt-4 bp-320:pt-4 md:pt-4 lg-pt-4 pr-4 bp-320:pl-2 h-3 bp-320:h-10 md:h-3 lg-h-3"
```

Comment:
`lg-pt-4` and `lg-h-3` are invalid

Reply:
instead of `h-3 bp-320:h-10`, you can simply write `h-10` as we are not focusing on screen sizes < 320 px